airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -0,0 +1,240 @@
+#
```

Comment:
the functions in this file should be relatively easy to unit test since it does not depend on anything else and has clear inputs and outputs. I think it would be valuable to do so since there's a lot of [non-trivial logic](https://github.com/airbytehq/airbyte/pull/32345/files#diff-576718ef29df03d12cb7402cc8c7b43c114ef03f5afad172854633aa8c294ca4R208-R240)